contracts/curation/Curation.sol

```diff
@@ -395,6 +456,8 @@ contract Curation is CurationV1Storage, GraphUpgradeable, ICuration {
         // Get curation pool tokens and signal
         CurationPool memory curationPool = pools[_subgraphDeploymentID];
 
+        uint32 _effectiveReserveRatio = _getEffectiveReserveRatio(curationPool.createdAt);
```

Comment:
Something that happens with this implementation is that it breaks the possibility to mint at the Curation contract level. 

If the Curation.mint() function is used directly by any user the `createdAt` variable will be zero. I think we should store `createdAt` on the subgraph struct whenever the first mint happens as part of initialization.